src/components/dialogs/network-modifications/two-windings-transformer/tap-changer-pane/ratio-tap-changer-pane/ratio-tap-changer-pane.js

```diff
@@ -271,19 +274,23 @@ const RatioTapChangerPane = ({
                 <Grid item container xs={4}>
                     {ratioTapLoadTapChangingCapabilitiesField}
                 </Grid>
-                <Grid item container spacing={2}>
-                    {isRatioTapLoadTapChangingCapabilitiesOn && (
+                {isRatioTapLoadTapChangingCapabilitiesOn && (
```

Comment:
Can you refactor it please ? The condition with `isRatioTapLoadTapChangingCapabilitiesOn` is already present 3 times below.

Reply:
Done